lerobot/configs/default.py

```diff
@@ -0,0 +1,217 @@
+    # AND for the evaluation environments.
+    seed: int | None = None
+    # You may provide a list of datasets here. `train.py` creates them all and concatenates them. Note: only data
+    # keys common between the datasets are kept. Each dataset gets and additional transform that inserts the
+    # "dataset_index" into the returned item. The index mapping is made according to the order in which the
+    # datsets are provided.
+    video_backend: str = "pyav"
+    training: TrainConfig = field(default_factory=TrainConfig)
+    eval: EvalConfig = field(default_factory=EvalConfig)
+    wandb: WandBConfig = field(default_factory=WandBConfig)
+
+    def __post_init__(self):
+        if not self.job_name:
+            self.job_name = f"{self.env.type}_{self.policy.type}"
+
+        if not self.dir:
+            now = dt.datetime.now()
+            train_dir = f"{now:%Y-%m-%d}/{now:%H-%M-%S}_{self.job_name}"
+            self.dir = Path("outputs/train") / train_dir
+
+        if self.training.online.steps > 0 and isinstance(self.dataset.repo_id, list):
+            raise NotImplementedError("Online training with LeRobotMultiDataset is not implemented.")
+
+        # If we are resuming a run, we need to check that a checkpoint exists in the log directory, and we need
+        # to check for any differences between the provided config and the checkpoint's config.
+        checkpoint_cfg_path = self.dir / "checkpoints/last/config.yaml"
+        if self.resume:
+            if not checkpoint_cfg_path.exists():
+                raise RuntimeError(
+                    f"You have set resume=True, but there is no model checkpoint in {self.dir}"
+                )
+
+            # Get the configuration file from the last checkpoint.
+            checkpoint_cfg = self.from_checkpoint(checkpoint_cfg_path)
+
+            # # Check for differences between the checkpoint configuration and provided configuration.
+            # # Hack to resolve the delta_timestamps ahead of time in order to properly diff.
+            # resolve_delta_timestamps(cfg)
+            diff = DeepDiff(checkpoint_cfg, self)
+            # Ignore the `resume` and parameters.
+            if "values_changed" in diff and "root['resume']" in diff["values_changed"]:
+                del diff["values_changed"]["root['resume']"]
+            # Log a warning about differences between the checkpoint configuration and the provided
+            # configuration.
+            if len(diff) > 0:
+                logging.warning(
+                    "At least one difference was detected between the checkpoint configuration and "
+                    f"the provided configuration: \n{pformat(diff)}\nNote that the checkpoint configuration "
+                    "takes precedence.",
+                )
+            # Use the checkpoint config instead of the provided config (but keep `resume` parameter).
+            self = checkpoint_cfg
```

Comment:
pretty sure this doesn't do what you want?

Reply:
Indeed that doesn't work at all, I will handle that part once I'm done with the rest of the config (I'm on the policies right now, which is quite a big chunk).
Thanks for the heads up!